aiohttp/web_response.py

```diff
@@ -506,27 +555,24 @@ def __init__(self, *,
         self._compressed_body = None
 
     @property
-    def body(self):
+    def body(self) -> Optional[Union[payload.Payload, bytes]]:
         return self._body
 
     @body.setter
-    def body(self, body,
-             CONTENT_TYPE=hdrs.CONTENT_TYPE,
-             CONTENT_LENGTH=hdrs.CONTENT_LENGTH):
+    def body(self,
+             body: Any,
+             CONTENT_TYPE: str=hdrs.CONTENT_TYPE,
+             CONTENT_LENGTH: str=hdrs.CONTENT_LENGTH) -> None:
         if body is None:
             self._body = None
-            self._body_payload = False
```

Comment:
So, it turns out that `_body` can be either `bytes` or `Payload`. Seemed more correct to rely on `isinstance` then keeping extra property in the class.